python/tests/gold/clientcertificateauthentication/bootstrap-ads.json

```diff
@@ -31,13 +31,8 @@
       {
         "name": "static_layer",
         "static_layer": {
-          "envoy.deprecated_features.allow_deprecated_gzip_http_filter": true,
-          "envoy.deprecated_features:envoy.api.v2.route.HeaderMatcher.regex_match": true,
-          "envoy.deprecated_features:envoy.api.v2.route.RouteMatch.regex": true,
-          "envoy.deprecated_features:envoy.config.trace.v2.ZipkinConfig.HTTP_JSON_V1": true,
```

Comment:
Hmmm, do we need to document the removal of these deprecated features anywhere other than the changelog? Thinking about the gzip and Zipkin ones specifically, but I cant say for sure if that is an issue for users.

Reply:
Great question...let me investigate these further. They were removed by aidan, so I followed suit. I do agree that we should document them.

Reply:
Do we still think we need to call these out beyond the removal V2 which was already added. I talk about it in the migration guide. https://github.com/datawire/getambassador.io/blob/a3cfb2147dea099d3ab61072fdba8943e4a5b41d/ambassador-docs/docs/edge-stack/pre-release/about/changes-3.y.md

Which needs some attention before release.